src/code/fv-components/fv-gamete-image.js

```diff
@@ -28,18 +28,19 @@ const FVGameteImageView = ({chromosomes=[], id, className, style, displayStyle})
     chromosomeImages.push(
       <ChromosomeImageView
         key={i}
-        empty={!chromosome}
-        width={6 * scale}
-        height={(chromosome && chromosome.side === "y" ? 15 : 20) * scale}
-        split={8 * scale}
+        empty={false}
```

Comment:
This is the only confusing bit- is the idea that empty sets the color to white, and we always want to use the colors passed below? Not that this is worth messing with now, but a passed color should probably override any flags, right?